validation/rally-zero-one/measurements.1.attention-example.pass.json

```diff
@@ -0,0 +1,18 @@
+{
+  "pageId": "7df069eef8472fc4d39b5c805a67f2f0",
+  "origin": "https://www.wikipedia.org",
+  "referrerOrigin": "",
+  "pageVisitStartTime": 1618243280228,
+  "pageVisitStopTime": 1618243283936,
+  "duration": 3460,
+  "maxRelativeScrollDepth": 0.5968342644320298,
+  "maxPixelScrollDepth": 641,
+  "scrollHeight": 1074,
+  "eventTerminationReason": "page-visit-stop",
+  "title": "Wikipedia",
+  "ogType": "",
```

Comment:
A few of these are empty but present. Are we sure they will always be present? Do we want to un-require them?

Reply:
Not sure, @hamilton what are your thoughts here?

Reply:
They will always be present. The data collection library we use sometimes reports empty values.